lib/generators/react_webpack_rails/install_generator.rb

```
@@ -1,28 +1,33 @@
 
-
     def generate_layout
       copy_file '.babelrc', '.babelrc'
+      copy_file 'initializers/react_webpack_rails.rb', 'config/initializers/react_webpack_rails.rb'
+      copy_file 'karma.conf.js', 'karma.conf.js'
+      copy_file 'node_server.js', 'node_server.js'
+      copy_file 'partial/_react_hot_assets.html.erb', 'app/views/layouts/_react_hot_assets.html.erb'
       copy_file 'webpack.config.js', 'webpack.config.js'
       copy_file 'webpack/dev.config.js', 'webpack/dev.config.js'
       copy_file 'webpack/hot-dev.config.js', 'webpack/hot-dev.config.js'
       copy_file 'webpack/production.config.js', 'webpack/production.config.js'
-      copy_file 'karma.conf.js', 'karma.conf.js'
       copy_file 'webpack/tests.config.js', 'webpack/tests.config.js'
-      copy_file 'partial/_react_hot_assets.html.erb', 'app/views/layouts/_react_hot_assets.html.erb'
-      template 'react/index.js.erb', 'app/react/index.js'
       create_file 'app/assets/javascripts/react_bundle.js'
+      template 'package.json.erb', 'package.json'
+      template 'react/index.js.erb', 'app/react/index.js'
       if options.example
-        copy_file 'react/components/hello-world.jsx', 'app/react/components/hello-world.jsx'
-        copy_file 'react/components/hello-world-test.jsx', 'app/react/components/hello-world-test.jsx'
+        copy_file 'example/controllers/rwr_examples_controller.rb', 'app/controllers/rwr_examples_controller.rb'
+        copy_file 'example/react/components/hello-world-test.jsx', 'app/react/components/hello-world-test.jsx'
+        copy_file 'example/react/components/hello-world.jsx', 'app/react/components/hello-world.jsx'
+        copy_file 'example/views/rwr_examples/client_side_hello_world.html.erb', 'app/views/rwr_examples/client_side_hello_world.html.erb'
+        route "get 'client_side_hello_world', to: 'rwr_examples#client_side_hello_world', as: :client_side_hello_world"
```

But then you won't get a fully-functional example, which anyways is behind a flag already. I can change the flag's default to false and hide node_server.js behind other flag (e.g. `node_server:true`).